init.lua

```diff
@@ -480,6 +480,12 @@ minetest.register_node("anvil:anvil", {
 	on_blast = function(pos, intensity)
 		local drops = {"anvil:anvil"}
 		local meta = minetest.get_meta(pos)
+		if meta:get_int("shared") == 1 then
+			drops[1] = ItemStack({
```

Comment:
Suggestion: Add a new function to construct this ItemStack because `shared_anvil_craft_stack` needs the same code below.